app/views/users/show.html.erb

```diff
@@ -1,4 +1,57 @@
-<div class="container">
-  <h2><%= @user.email %>'s Profile</h2>
+<div class="">
```

Comment:
Doesn't bootstrap usually require a `container`/`container-fluid` style to house all its rows and columns?